docs/slack-connector/README.md

```diff
@@ -7,4 +7,4 @@ The Slack Connector is an additional tool to Kyma for handling communication wit
 ## Table of Contents
```

Comment:
**Line 5:**
  - I'd say `an additional tool for Kyma` or `an addon to Kyma`
  - `Slack API` -> `the Slack API`